code/parachain/frame/crowdloan-rewards/src/lib.rs

```diff
@@ -350,6 +381,54 @@ pub mod pallet {
 	}
 
 	impl<T: Config> Pallet<T> {
+		/// Patch existing rewards by deleting them.
+		/// This will cause inconsistencies if a claim has already been made. Errors can be avoided
+		/// by passing the expected amount of rewards claimable.
+		///
+		/// Security:
+		/// * Should be called within a transactional context, as this may return an error but still
+		///   change
+		/// storage.
+		pub(crate) fn do_delete(
```

Comment:
NIT: We may comment this out as well not to have this affect the audits

Reply:
Agreed, we also want to keep the CTO at the bottom of contributors.